### PR TITLE
Add SIG Node charter 

### PR DESCRIPTION
**Pull request by Shelley-BaoYue**
Signed-off-by: Shelley-BaoYue <baoyue2@huawei.com>

add:
/sig-node

### CODE REVIEW

File: sig-node/charter.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#### Areas of Focus
+
+- Node lifecycle management
+- Kubelet and its feature. Kubelet is integrated into edged mode to manage containerized applications.
+- Extremely lightweight with lite-Kubelet
+- Node level performance and scalability
+- Node reliability
+- Node-level resource management and monitoring information collection for hosts at the edge, such as power supply, battery power, etc.
+- Nodegroup and edgeapplication management
+- Communication across node, such as node dynamic trunk. (with [EdgeMesh](https://github.com/kubeedge/edgemesh))
```
Comment (benjaminhuo) [inline sig-node/charter.md:20]:
```suggestion
- Communication across nodes, such as node dynamic trunk. (with [EdgeMesh](https://github.com/kubeedge/edgemesh))
```

**Review by benjaminhuo — CHANGES_REQUESTED** [review]

File: sig-node/charter.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# SIG Node Charter
+
+This charter adheres to the conventions described in the [KubeEdge Open Governance](https://github.com/kubeedge/community/blob/master/GOVERNANCE.md) and used the Roles and Organization Management outlined in the governance doc.
+
+## Scope 
+
+The SIG Node is responsible for the component that support the controlled interactions between pods and host resources. We manager the lifecycle of node and pods scheduled to the node. We focus on enabling a broad set of workload types, including workloads with hardware specific or performance sensitive requirements.  We support extremely lightweight edge agent(EdgeCore)  to run on resource constrained edge. Due to the complex scenario at edge, we manage the local metadata to support edge autonomy. We also aim to continuously improve node performance and reliability.
```
[the remark this answers is not in the document]
Comment (Shelley-BaoYue) [reply sig-node/charter.md]: > 

Done.

File: sig-node/charter.md (hunk 1 above)
Comment (Shelley-BaoYue) [reply sig-node/charter.md]: > 

fixed.

**Review by fisherxu — COMMENTED** [review]
/lgtm

**Pull request merged**